includes/class-wc-calypso-bridge-ecom-free-trial-frontend.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Class WC_Calypso_Bridge_ECOM_Free_Trial_Frontend.
+ *
+ * @since   1.9.16
+ * @version 1.9.16
+ *
+ * Handles Free Trial frontend.
+ */
+class WC_Calypso_Bridge_ECOM_Free_Trial_Frontend  {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+	
+	public function __construct(){
```

Comment:
Should we add this notice for shop managers and administrators, only? 🤔

Reply:
Thank you for the detailed feedback @xristos3490 👍 

> Should we add this notice for shop managers and administrators, only?

That's a good point! I think the notice should be for just shop managers and administrators.

I'll confirm with design team and make a change 👍

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/917/commits/e57816c1ccfa263b0984fce9455c4dfc516c8c0e